api/DHSC.FingertipsNext.Api/Controllers/HealthMeasureController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DHSC.FingertipsNext.Modules.Core.Service;
```

Comment:
This controller should be in a module rather than at the top level.  Rather than using the Core module throughout, you'll need to put it in the HealthData module but that is currently in a different PR that hasn't been merged yet.  Lets discuss how to get around this.

Reply:
This controller is essentially a placeholder until that PR is in so more than happy to bin / move things as required